akka-stream/src/main/scala/akka/stream/scaladsl/Framing.scala

```diff
@@ -34,7 +34,7 @@ object Framing {
    *                           exceeded this Flow will fail the stream.
    */
   def delimiters(
-      delimiters: List[ByteString],
+      delimiters: Seq[ByteString],
```

Comment:
Now it accepts mutable sequences as well, at least on Scala 2.12, should be specifically `scala.collection.immutable.Seq`, we usually import `scala.collection.immutable` and use `immutable.Seq` to make that extra explicit in the APIs.